src/main/scala/com/horizon/exchangeapi/OrgsRoutes.scala

```diff
@@ -327,6 +340,69 @@ trait OrgsRoutes extends JacksonSupport with AuthenticationSupport {
       }) // end of complete
     }   // end of exchAuth
   }
+  // ====== GET /orgs/{orgid}/status ================================
+  @GET
+  @Path("orgs/{orgid}/status")
+  @Operation(summary = "Returns an org specific status info", description = "Returns the org with the specified id. Can be run by any user in this org or a hub admin.",
+    parameters = Array(
+      new Parameter(name = "orgid", in = ParameterIn.PATH, description = "Organization id.")),
+    responses = Array(
+      new responses.ApiResponse(responseCode = "200", description = "response body",
+        content = Array(new Content(schema = new Schema(implementation = classOf[GetOrgStatusResponse])))),
+      new responses.ApiResponse(responseCode = "401", description = "invalid credentials"),
+      new responses.ApiResponse(responseCode = "403", description = "access denied"),
+      new responses.ApiResponse(responseCode = "404", description = "not found")))
+  def orgStatusRoute: Route = (path("orgs" / Segment /"status") & get ) { (orgId) =>
+    exchAuth(TOrg(orgId), Access.READ) { ident =>
+      logger.debug(s"GET /orgs/$orgId/status")
+      complete({
+        val statusResp = new OrgStatus()
+        //perf: use a DBIO.sequence instead. It does essentially the same thing, but more efficiently
+        db.run(UsersTQ.rows.length.result.asTry.flatMap({
```

Comment:
The way all of your DB queries are right now, they return the number of that type of resource for the **whole** exchange. This route needs to return the number of that type of resource in this org. By convention, we put most of those queries in the `*TQ` object. But for a lot of what you need, there already is an appropriate query in there. So, for example, in `tables/Users.scala` there is already a method `UsersTQ.getAllUsers(orgid)` that gets the users in this org (see the filter). So you just have to turn it into the number of users with this: `UsersTQ.getAllUsers.length.result`

This comments also applies to all of the other queries, except for the db schema version.